.travis.yml

```diff
@@ -124,12 +124,64 @@ matrix:
       - cd kitchen-tests
     script:
       # FIXME: we should fix centos-6 against AWS and then enable it here
-      - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen test ubuntu; fi
+      - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen test ubuntu-1404 -l debug; fi
     after_failure:
       - cat .kitchen/logs/kitchen.log
     after_script:
-      - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen destroy ubuntu; fi
+      - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen destroy ubuntu-1404 -l debug; fi
```

Comment:
these two scripts have `-l debug`, but the separate centos block below does not.